pkg/volume/util/atomic_writer.go

```diff
@@ -125,7 +124,7 @@ func (w *AtomicWriter) Write(payload map[string]FileProjection) error {
 	}
 
 	// (2)
-	dataDirPath := path.Join(w.targetDir, dataDirName)
+	dataDirPath := filepath.Join(w.targetDir, dataDirName)
```

Comment:
@feiskyer can you review these changes from windows perspective?

Reply:
yep, of course